src/video/windows/SDL_windowsevents.c

```diff
@@ -490,6 +490,22 @@ WIN_KeyboardHookProc(int nCode, WPARAM wParam, LPARAM lParam)
     return 1;
 }
 
+static void UpdateFocus(HWND hwnd, SDL_Window * window)
+{
+    if (SDL_GetKeyboardFocus() != window) {
```

Comment:
In line with my query in the PR description, I'd like confirmation whether this check should be here. The alternative is reversing the order of checks and updating `SetKeyboardFocus` with the corrected values.